### DISCUSSION TITLE
Fix lag on loading many commit suggestions

### DISCUSSION THREAD

**Discussion by matkoniecz** [community/community#11839] [Pull Requests]
For example, opening https://github.com/streetcomplete/StreetComplete/pull/3648/files#commit-suggestions lags for several seconds after opening.

What worse, applying commit suggestions results in page reloading and spinning for several seconds again.

As workaround I closed PR and reopened as https://github.com/streetcomplete/StreetComplete/pull/3789

**Comment by rugk** [comment]
Did you try out GitHub's new tree view on PRs? Maybe this helps?

See https://github.com/github/feedback/discussions/12341

**Reply by matkoniecz** [reply]
There is still a noticeable lag, but seems much better
